index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Create Mobile Version</title>
+</head>
+<body>
+    <main>
+        
+        <header>
+            <h1 class="logo">Welcome</h1>
+     
+            <nav class="menu-bar">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </nav>
+
+        </header>
+
+        <div class="background-image">
+          <div class="about-me">
+             
+                <p class="intro">Hey there. I’m Dozie <br><span class="who-am-i">I’m a software developer</span></p>
+                
+             <div>
+                <p class="i-can-do"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+             </div>
+                
+
+              
+          </div>
+                    
+                <ul class="social">
+                    <li class="social-acc"><img src="asset/images/twitter.png" alt="twitter"></li>
+                    <li class="social-acc"><img src="asset/images/Linkedin-icon.png" alt="Linkedin"></li>
+                    <li class="social-acc"><img src="asset/images/medium.png" alt="email"></li>
+                    <li class="social-acc"><img src="asset/images/github.jpg" alt="github"></li>
+                    <li class="social-acc"><img src="asset/images/angellist.jpg" alt="angellist"></li>
```

Comment:
- [ ] You should wrap the social media icons with an `<a>` so that they become clickable.  Example: 
```
<li> 
    <a href="#">
     <img src="asset/images/github.jpg" alt="github">
   </a>
</li>
```

Reply:
The social media icons have been rapped appropriately.